custom_components/tuya_local/devices/smart_camera_sc116-wz3a.yaml

```diff
@@ -0,0 +1,511 @@
+name: Camera
+products:
+  - id: b60fclvud3xtcufm
+    manufacturer: Tuya
+    model: SC116-WZ3A
+entities:
+  - entity: camera
+    dps:
+      - id: 105
+        type: boolean
+        optional: true
+        name: switch
+        mapping:
+          - dps_val: true
+            value: false
+          - dps_val: false
+            value: true
+          - dps_val: null
+            value: true
+            hidden: true
+      - id: 134
+        type: boolean
+        name: motion_enable
+        optional: true
+      - id: 150
+        type: boolean
+        name: record
+        optional: true
+      - id: 115
+        type: base64
+        optional: true
+        persist: false
+        sensitive: true
+        name: snapshot
+  - entity: switch
+    name: Indicator
```

Comment:
This should be a light entity, then there is a translation_key for indicator to use instead of name and icon.